test/cli/config-cwd-case-insensitive/_config.js

```diff
@@ -0,0 +1,13 @@
+var os = require( 'os' );
+
+function toggleCase ( s ) {
+  return ( s == s.toLowerCase() ) ? s.toUpperCase() : s.toLowerCase();
+}
+
+module.exports = {
+  skip: os.platform() !== 'win32',
```

Comment:
The test is specific to Windows.  I can remove the `skip` and it should still pass on OSX and Linux, but it's meaningless.

Can you be more specific about what indentation?  I guess maybe placing all of a ternary statement on a single line?  Where can I find the Rollup coding style guidelines?